packages/core/bootstrap/test/unit/cache-key.test.ts

```diff
@@ -47,15 +47,14 @@ describe('Cache key middleware', () => {
                   batchPropString: 'batchString',
                   endpoint: 'random',
                   from: 'btc',
-                  includes: {},
                   maxAge: 444,
                   number: 123.4,
                   overrides: {
-                    btc: 'bitcoin',
+                    btc: { bitcoin: 'bitcoin' },
                   },
                   to: 'eth',
                   tokenOverrides: {
-                    eth: 'test',
+                    eth: { test: 'test' },
```

Comment:
While the way you have changed it is still an accurate test, let's structure these how they should actually look

```suggestion
          adapterName: { eth: 'test' },
```